edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup Condition="'$(DotNet_Runtime)' != 'netcoreapp3.0'">
+        <TargetFramework>netcoreapp2.1</TargetFramework>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(OS)|$(DotNet_Runtime)' == 'Unix|netcoreapp3.0'">
+        <TargetFramework>netcoreapp3.0</TargetFramework>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+        <Configurations>Debug;Release;CodeCoverage;CheckInBuild</Configurations>
+        <HighEntropyVA>true</HighEntropyVA>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="CliWrap" Version="2.4.0" />
+        <PackageReference Include="FluentAssertions" Version="5.9.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+        <PackageReference Include="Nito.AsyncEx" Version="5.0.0" />
```

Comment:
interesting that i found that edge.util project currently use 5.0.0-pre-05; please try to use 1 version only, either update existing to use 5.0.0 or update your project to use preview version.

Reply:
Agree